### PR TITLE
add new services

### PR DESCRIPTION
**Pull request by madebyTimo**
add keycloak and FiniteQuiz.

Part of Gamify-IT/issues#252

### CODE REVIEW

File: docker-compose-test.yaml [hunk 1]
```diff
@@ -163,6 +201,13 @@ services:
     expose:
       - "80"
 
+  finitequiz-test:
+    container_name: finitequiz-test
+    image: ghcr.io/gamify-it/finitequiz:main
+    restart: always
+    expose:
+      - "80"
+
```
Comment (Gr33ndev) [inline docker-compose-test.yaml:210]:
```suggestion
 third-party-license-notice-test:
    container_name: third-party-license-notice-test
    image: ghcr.io/gamify-it/third-party-license-notice:main
    restart: always
    expose:
      - "80"
```

File: docker-compose.yml [hunk 2]
```diff
@@ -163,6 +200,13 @@ services:
     expose:
       - "80"
 
+  finitequiz:
+    container_name: finitequiz
+    image: ghcr.io/gamify-it/finitequiz:latest
+    restart: always
+    expose:
+      - "80"
+
```
Comment (Gr33ndev) [inline docker-compose.yml:209]:
```suggestion
 third-party-license-notice:
    container_name: third-party-license-notice
    image: ghcr.io/gamify-it/third-party-license-notice:latest
    restart: always
    expose:
      - "80"
```

File: nginx/conf.d-test/default.conf [hunk 3]
```diff
@@ -59,6 +64,11 @@ server {
...
         proxy_pass      http://bugfinder-test/;
     }
+
+    location /minigames/finitequiz/ {
+        proxy_pass      http://finitequiz-test/;
+    }
+
     location /lecturer-interface/ {
         proxy_pass      http://lecturer-interface-test/;
     }
```
Comment (Gr33ndev) [inline nginx/conf.d-test/default.conf:74]:
```suggestion
    }
    
    location /third-party-license-notice/ {
        proxy_pass      http://third-party-license-notice/;
    }
```

File: nginx/conf.d/default.conf [hunk 4]
```diff
@@ -60,6 +65,10 @@ server {
         proxy_pass      http://bugfinder/;
     }
 
+    location /minigames/finitequiz/ {
+        proxy_pass      http://finitequiz/;
+    }
+
     location /lecturer-interface/ {
         proxy_pass      http://lecturer-interface/;
     }
```
Comment (Gr33ndev) [inline nginx/conf.d/default.conf:74]:
```suggestion
    }
    
    location /third-party-license-notice/ {
        proxy_pass      http://third-party-license-notice/;
    }
```

**Review by Gr33ndev — CHANGES_REQUESTED** [review]

File: nginx/conf.d/default.conf (hunk 4 above)
Comment (madebyTimo) [reply nginx/conf.d/default.conf:74]: oh sorry i forgot it enen though you remembered me so often :(

**Review by Gr33ndev — APPROVED** [review]

**Review by robojones — APPROVED** [review]

**Pull request merged**